apps/gallery/index.html

```diff
@@ -109,11 +109,11 @@ <h1 data-l10n-id="gallery" id="fullscreen-title">Gallery</h1>
       </section>
       <button id="fullscreen-back-button-tiny" type="reset" class="tiny-only" data-icon="back"></button>
       <footer id="fullscreen-toolbar" class="tiny-only">
-        <a id="fullscreen-camera-button-tiny" class="button" data-icon="camera"></a>
-        <a id="fullscreen-edit-button-tiny" class="button" data-icon="edit-image"></a>
-        <a id="fullscreen-share-button-tiny" class="button" data-icon="share"></a>
-        <a id="fullscreen-info-button-tiny" class="button" data-icon="info"></a>
-        <a id="fullscreen-delete-button-tiny" class="button" data-icon="delete"></a>
+        <a id="fullscreen-camera-button-tiny" class="button" data-icon="camera" data-l10n-id="camera-button" role="button"></a>
+        <a id="fullscreen-edit-button-tiny" class="button" data-icon="edit-image" data-l10n-id="edit-button" role="button"></a>
+        <a id="fullscreen-share-button-tiny" class="button" data-icon="share" data-l10n-id="share-button" role="button"></a>
+        <a id="fullscreen-info-button-tiny" class="button" data-icon="info" data-l10n-id="info-button" role="button"></a>
+        <a id="fullscreen-delete-button-tiny" class="button" data-icon="delete" data-l10n-id="delete-button" role="button"></a>
```

Comment:
From accessibility stand point above change works. Ideal is above links updated to real element 'button' , than we don't need role="button" explicitly specified. Please check if it's a simple change, else it can be done as separate bug as part of refactoring. Thanks